packages/askui-nodejs/src/core/annotation/annotation-json.ts

```diff
@@ -2,5 +2,5 @@ import { DetectedElement } from '../model/annotation-result/detected-element';
 
 export interface AnnotationJson {
   image: string,
-  objects: DetectedElement[]
+  data: DetectedElement[]
```

Comment:
not sure about the renaming, let's wait for @adi-wan-askui or @programminx-askui  review

Reply:
to be consistent with the new name of the array in the backend.

Reply:
but the old response should be now inside data, which means objects will be part of the new response.